doc/manipulating_images/index.rst

```diff
@@ -0,0 +1,21 @@
+.. include:: ../tune_toc.rst
+
+
+.. _manipulating_images:
```

Comment:
I would call this link label "image_manipulation", to be closer to the title that it represents.